CRM/MembershipExtras/Helper/MembershipTypeSettings.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+class CRM_MembershipExtras_Helper_MembershipTypeSettings {
```

Comment:
We have a class we use to manage settings, might be a good idea to merge this class with that one:

https://github.com/compucorp/uk.co.compucorp.membershipextras/blob/04060958a520b6198258175eb5ff97ccca00cc38/CRM/MembershipExtras/SettingsManager.php#L6